### PR TITLE
feat: Improve form item and switch to README preview on validation check

### PR DESCRIPTION
**Pull request by agatha197**
**Changes:**

This PR updates the ImportFromHuggingFaceModal component and related translations to enhance the user experience when importing models from Hugging Face. Key changes include:

1. Added new form fields for "Model store folder name" and "Service name".
2. Implemented a README.md preview card with Markdown support.
3. Added validation to ensure only LLM models (text-generation pipeline) are supported.
4. Updated translations for new strings across multiple languages.

**Rationale:**

These changes provide users with more control over the import process and better information about the model being imported. The README preview allows users to review model details before importing, while the additional form fields enable customization of the import location and service name.

**Effects:**

- Users can now specify custom folder names and service names for imported models.
- The UI now displays a preview of the model's README, improving user understanding of the model before import.
- The system now restricts imports to LLM models only, preventing potential issues with unsupported model types.
- Improved localization support for the new features across multiple languages.

**Screenshots:**
- Default

- Valid

- Invalid URL

- Not LLM model

**Checklist:**

- [ ] Documentation
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by agatha197** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2715?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2713** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2713?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2715** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2715?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2710** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2710?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2709** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2709?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2514** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2514?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @agatha197 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: react/src/components/ImportFromHuggingFaceModal.tsx [hunk 1]
```diff
@@ -124,7 +158,10 @@ const ImportFromHuggingFaceModal: React.FC<ImportFromHuggingFaceModalProps> = ({
           type="primary"
           htmlType="submit"
           onClick={handleOnClick}
-          disabled={!shouldSkipURLCheck}
+          disabled={
+            !shouldSkipURLCheck ||
+            huggingFaceModelInfo.data?.pipeline_tag! !== 'text-generation'
+          }
```
Comment (yomybaby) [inline react/src/components/ImportFromHuggingFaceModal.tsx]:
```suggestion
          disabled={
            !shouldSkipURLCheck ||
            (!_.isEmpty(huggingFaceModelInfo.data?.pipeline_tag) &&
              huggingFaceModelInfo.data?.pipeline_tag !== 'text-generation')
          }
```

**Review by yomybaby — CHANGES_REQUESTED** [review]

File: react/src/components/ImportFromHuggingFaceModal.tsx (hunk 1 above)
Comment (yomybaby) [reply react/src/components/ImportFromHuggingFaceModal.tsx]: I just push a change to resolve this part. LGTM now!

**Review by yomybaby — APPROVED** [review]
LGTM

**Pull request merged**